pyproject.toml

```diff
@@ -17,7 +17,7 @@ dependencies = [
     "packaging",
     "pint",
     "bluesky>=1.13.0a3",
-    "event-model<1.21.0",
```

Comment:
If DLS needs this pinned somehow for now, to keep the original draft of StreamResource, could it be pinned in a DLS-specific package? Maybe dodal?